include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -662,6 +683,23 @@ MGTransferGlobalCoarsening<dim, VectorType>::interpolate_to_mg(
     this->transfer[l].interpolate(dst[l - 1], dst[l]);
 }
 
+
+
+template <int dim, typename VectorType>
+std::size_t
+MGTransferGlobalCoarsening<dim, VectorType>::memory_consumption() const
+{
+  std::size_t size = 0;
+
+  const unsigned int min_level = transfer.min_level();
+  const unsigned int max_level = transfer.max_level();
+
+  for (unsigned int l = max_level; l > min_level; --l)
+    size += this->transfer[l].memory_consumption();
```

Comment:
Not wrong though maybe uncommon to count down :-)

Reply:
> Not wrong though maybe uncommon to count down :-) 

I just copied the code from above.

> You declare three new functions, but implement only two. There must be a missing one :-)

This is fine as it is. `MGTwoLevelTransfer<dim, LinearAlgebra::distributed::Vector<Number>>` is the only class specialization of `MGTwoLevelTransfer`.